src/overlays/actors/ovl_En_Ru2/z_en_ru2_cutscene_data.inc.c

```diff
@@ -2,7 +2,7 @@
 #include "z64cutscene_commands.h"
 
 // clang-format off
-static CutsceneData D_80AF411C[] = {
+static CutsceneData gWaterMedallionCS[] = {
```

Comment:
nit: `gWaterMedallionCs` is more consistent with existing names (e.g. `gIceCavernSerenadeCs`)

Reply:
The rest of the named CS data variables in `csdis_re.py` end in `CS`, not `Cs`.

Reply:
That is a tool for processing assets, external to the games code. When considering naming conventions of code for the game we are speaking within the context of `src/` and related header files

Reply:
Oh sorry, I see what you are saying. The symbol names within `HARDCODED_SYM_ROM`.
Those should change to conform to rest of the codebases style (also what is this script even doing?)

Reply:
I didn't realize the naming was inconsistent sorry, disregard